### PR TITLE
Added Unbound graph support

### PR DESCRIPTION
**Pull request by alangregory**
#### Please note

> Please read this information carefully.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

- Added Unbound graph support, missing unix-agent script for now.

### CODE REVIEW

File: includes/polling/applications/unbound.inc.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+}
+#Format Data
+$lines = explode("\n",$rawdata);
+$unbound = array();
+foreach ($lines as $line) {
+	list($var,$value) = explode('=',$line);
+	$unbound[$var] = $value;
+}
+#Unbound Queries
+$rrd_name =  array('app', $name."-queries",$app_id);
```
Comment (murrant) [inline includes/polling/applications/unbound.inc.php]: this uses the rrd_name() function,
You could just use:
```
$rrd_name =  array('app', $name, 'queries', $app_id)
```

**Comment by murrant** [comment]
Thanks!  You will need to add some thing to the Documentation as well.

File: includes/polling/applications/unbound.inc.php (hunk 1 above)
Comment (alangregory) [inline includes/polling/applications/unbound.inc.php]: Thanks for pointing out.

**Comment by alangregory** [comment]
I'll add docs as soon as i finish the unix-agent script.

File: doc/Extensions/Applications.md [hunk 2]
```diff
@@ -168,6 +168,24 @@ extend memcached /usr/local/bin/memcached
...
+[Install the agent](#agent-setup) on this device if it isn't already and copy the `unbound.sh` script to `/usr/lib/check_mk_agent/local/`
+
+Unbound configuration:
+
+```text
+# Enable extended statistics.
+server:
+        statistics-interval: 0
+        extended-statistics: yes
+        # set to yes if graphing tool needs it
```
Comment (murrant) [inline doc/Extensions/Applications.md]: Why is this comment here?

File: scripts/unbound.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+unboundctl=`which unbound-control`
```
Comment (murrant) [inline scripts/unbound.sh]: We have been putting these scripts in the librenms-agent repo.

File: includes/polling/applications/unbound.inc.php [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+	'naptr' => $unbound['num.query.type.NAPTR'],
+	'ds' => $unbound['num.query.type.DS'],
+	'dnskey' => $unbound['num.query.type.DNSKEY'],
+	'spf' => $unbound['num.query.type.SPF'],
+	'any' => $unbound['num.query.type.ANY'],
+	'other' => $unbound['num.query.type.other']
+	);
+$tags = compact('name', 'app_id', 'rrd_name', 'rrd_def');
+data_update($device, 'app', $tags, $fields);
+$status = true;
```
Comment (murrant) [inline includes/polling/applications/unbound.inc.php]: Unused variable?

File: doc/Extensions/Applications.md (hunk 2 above)
Comment (alangregory) [inline doc/Extensions/Applications.md]: It's from the oficial guide [Unbound: How to Statistics ](https://www.unbound.net/documentation/howto_statistics.html), fixed on 8e7d40c

File: includes/polling/applications/unbound.inc.php (hunk 4 above)
Comment (alangregory) [inline includes/polling/applications/unbound.inc.php]: Fixed on 9de3d43

File: scripts/unbound.sh (hunk 3 above)
Comment (alangregory) [inline scripts/unbound.sh]: Removed here 3ec0b94 and moved to librenms/librenms-agent/pull/50

**Pull request merged**